tfx/components/example_validator/component.py

```diff
@@ -95,6 +99,9 @@ def __init__(self,
         type=standard_artifacts.ExampleAnomalies,
         artifacts=[standard_artifacts.ExampleAnomalies()])
     spec = ExampleValidatorSpec(
-        statistics=statistics, schema=schema, anomalies=anomalies)
+        statistics=statistics,
+        schema=schema,
+        exclude_splits=exclude_splits,
+        anomalies=anomalies)
```

Comment:
you need to update executor too, it's currently hardcode a split I think, same for schema_gen executor

Reply:
yes i am updating executor, i will change the hardcode